test/fixtures/.lounge/config.js

```diff
@@ -3,6 +3,7 @@
 var config = require("../../../defaults/config.js");
 
 config.prefetch = true;
+config.prefetchStorage = false;
```

Comment:
Pretty pleeeease 🙏 :)
Something like [this line](https://github.com/thelounge/lounge/blob/master/test/plugins/link.js#L161) could check the format of the link, no? And with our current tests, we'd still have coverage for when the setting is disabled.
Now you'd have to be able to run tests with different config files, I don't know if any of our test suite would need refactor for that.